repose-aggregator/components/filters/regex-rbac-filter/src/main/resources/META-INF/schema/config/regex-rbac.xsd

```diff
@@ -0,0 +1,145 @@
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:complexType name="DelegatingType">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>Whether or not you would like this filter to populate the delegation headers. Inclusion means you do.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:attribute name="quality" type="QualityType" use="optional" default="0.3">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        What quality you want any output headers to be.
+                        When setting up a chain of delegating filters the highest quality number will be the one that is eventually output.
+                        Default is 0.3
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+        <xs:attribute name="component-name" type="xs:string" use="optional" default="regex-rbac">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Sets the delegation component name for this validator.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:simpleType name="QualityType">
+        <xs:restriction base="xs:double">
+            <xs:minInclusive value="0"/>
+            <xs:maxInclusive value="1.0"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:complexType name="ResourcesType">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    This is a list of resources, HTTP methods, and roles allowed to access them.
+
+                    The special HTTP methods ANY and ALL are shorthand for the four standard REST methods
+                    (GET, PUT, POST, DELETE). Other methods are supported, but they must be explicitly stated.
+                    If a given Resource line has the HTTP method of ANY/ALL, then only the four are utilized.
+                    If more methods need to be supported by a particular Resource, then put them on another Resource
+                    line.
+
+                    The RegEx RBAC resources can be either linked via an href attribute or defined directly in this
+                    element.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:simpleContent>
+            <xs:extension base="ResourcesString">
+                <xs:attribute name="href" type="xs:anyURI" use="optional">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>
+                                Specifies a location to an external file which contains the RegEx RBAC resources.
+                                If the message element has a value and the href attribute is configured,
```

Comment:
message element?

Reply:
Copy/Paste. Should be `resources`.
Updated.